mlbusinesscomponents/src/main/java/com/mercadolibre/android/mlbusinesscomponents/components/pickup/SizeType.kt

```diff
@@ -0,0 +1,33 @@
+package com.mercadolibre.android.mlbusinesscomponents.components.pickup
+
+import com.mercadolibre.android.mlbusinesscomponents.R
+import java.lang.Exception
+
+enum class SizeType(val fontSize: Int, val imageSize: Int) {
+    SMALL(R.dimen.ui_fontsize_xsmall, R.dimen.ui_2m),
+    MEDIUM(R.dimen.ui_fontsize_small, R.dimen.ui_image_size_medium);
+
+    companion object {
+        @JvmStatic
+        fun getFontSizeOrDefault(size: String?, defaultSize: Int) : Int {
+            size?.let {
+                return if (values().any { it.name.equals(size, ignoreCase = true) }) {
+                    valueOf(size).fontSize
+                } else {
+                    defaultSize
+                }
+            } ?: return defaultSize
```

Comment:
Ya que hacés un any debería hacer hacer un first en realidad, y se puede acomodar un poco para tener un solo return
```suggestion
                return size
                    ?.let { values().firstOrNull { value -> value.name.equals(it, ignoreCase = true) } }
                    ?.fontSize
                    ?: defaultSize
```